crates/holochain_sqlite/src/sql/p2p_agent_store/prune.sql

```diff
@@ -1,5 +1,30 @@
--- delete all expired entries from the p2p_agent_store
+-- it's hard to construct queries with input arrays
+-- we're taking the strategy here of concatonating a bunch of
+-- local agent ids together, and extracting them here
+WITH RECURSIVE split(src, cur_idx, slice) AS (
+  -- we have to manually configure the first seed row
+  SELECT
+    :agent_list,
+    37,
```

Comment:
What is 37 and 36?

Reply:
alas, substr in sqlite is 1-based, the 37 is saying, for the second round of the recursive function, we need to start at byte 37, and the 36 is the length, because the agent pubkeys are 36 bytes long.